lib/screens/listOfOrgs/listOfOrgsScreen.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
```

Comment:
Please follow the naming conversion...
1. file and folder names: `lower_case_words_separated_by_underscores`
2. class names: `UpperCamelCase`
3. variables and functions: `lowerCamelCase`